lib/fhir/query-utils.js

```diff
@@ -420,35 +420,13 @@ module.exports = (mongo) => {
       return clause
     },
 
-    extensionToMongoClause: (paramConf, parameters, modifier) => {
-      const buildQuery = (parameter) => {
-        const clause = {
-          extension: {
-            $elemMatch: { url: paramConf.url }
-          }
-        }
-
-        switch (modifier) {
```

Comment:
These would now be handled the same way normal query params are instead of separately like we had before. This code wasn't used anymore so it's safe to remove.